lib/repository_conversion_worker.rb

```diff
@@ -0,0 +1,10 @@
+class RepositoryConversionWorker < BaseWorker
+  sidekiq_options retry: false, queue: 'default'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.